docs/navigation-configuration.md

```diff
@@ -0,0 +1,120 @@
+## Overview
+
+Navigation parameters allow you to specify routing configuration, set the appearance of navigation, and define navigation structure. 
+
+## A basic example
+
+When navigating between nodes that are located on the same domain, Luigi triggers a hash or path change. Then, it sends an updated context in order not to fully reload the view. This is to the advantage of a single-page application based micro frontend. Navigation between domains triggers a full page load in order to comply with cross domain security concepts.
+
+This image shows where the how the Luigi navigation is positioned.  There is 
+
+- A top navigation
+- A side navigation
+- A main content window
+
+The image shows where you can use various methods and parameters to fill the navigation or display a micro frontend.  
+
+![Navigation layout](assets/navigation-structure.png)
+
+This code sample demonstrates your options when configuring navigation for Luigi. 
+
+````
+window.Luigi.setConfig({
+  routing: {
+    // uses hash based navigation if set to true
+    useHashRouting: true,
+    nodeParamPrefix: '~'
+  }
+  // navigation structure and settings
+  navigation: {
+    nodes: [
+        // STATIC navigation node
+      {
+        pathSegment: 'settings',
+        label: 'Settings',
+        viewUrl: 'https://admin.mydomain.com/settings',
+        // optional
+        children: [node, node, node],
+        hideFromNav: false,
+      },
+        // DYNAMIC navigation node
+      {
+        navigationContext: 'contextName',
+        pathSegment: project.id,
+        context: {
+          currentProject: project.id
+        }
+        children: [node, node, node]
+      }
+    ]
+  }
+});
+````
+### Routing
+
+- **useHashRouting** - Defines either hash-based or path-based routing. Hash-based routing uses the classic shebang For example, `url.com/!#/yourpath`. Path-based routing is uses the common path structure. For example, ` url.com/yourpath`. 
+- **nodeParamPrefix** - The function `LuigiClient.linkManager().withParam()`, found in the Luigi client, provides a way to simply attach query parameters to the view URL for activities such as sorting and filtering.  The URL contains the parameters to allow deep linking. If you want to use a different character prefix, define yours here. The default character is `~`.
+
+### Navigation parameters
+
+- **pathSegment** - Specifies the partial URL of the current segment. A static settings example would reflect  luigidomain.test/settings, while a dynamic one would get loaded on any other value.
```

Comment:
1. Turn the bullet points into full sentences.
2. ``A static settings example reflects `luigidomain.test/settings`, while a dynamic one loads on any other value.``